local-dev/run_postgres.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Start up a postgres container with initial user/database setup.
+POSTGRES_VERSION=9.6
+
+set -ex
+
+start() {
+    echo "attempting to remove old $CONTAINER container..."
+    docker rm -f $CONTAINER || echo "docker rm failed. nothing to rm."
```

Comment:
Since this is expected to fail, I'm inclined not to print anything if it does.

```suggestion
    docker rm -f $CONTAINER
```